contracts/badger-bridge/Zap.sol

```diff
@@ -0,0 +1,325 @@
+pragma solidity 0.6.11;
+
+import {IERC20} from "deps/@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20, SafeMath} from "deps/@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+//import "deps/@openzeppelin/contracts/token/ERC20/IERC20.sol";
+//import "deps/@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+//import {AccessControlDefended} from "./AccessControlDefended.sol";
```

Comment:
the AccessControlDefended is a contract that stores permissions for contracts to access other contracts - I initially disabled it just because I didn't feel like adding the different contracts that it requires. Is this sort of permissioning necessary for our zap contract?

Reply:
Nope, we should remove unused dependencies. Let's rename it to something else as we should remove the actual zap/deposit functionality since we're using this only to estimate optimal path. Wrote a nit below.